docs/src/pages/components/list-view.md

```diff
@@ -2,6 +2,12 @@
 title: QasListView
 ---
 
+<div class="flex q-gutter-x-md">
+  <doc-link title="Componente" name="QasDebugger" to="/components/debugger" />
+  <doc-link title="Componente" name="QasSingleView" to="/components/single-view" />
```

Comment:
faltou q-pull-to-refresh, q-inner-loading, q-pagination, q-spinner

Reply:
feito